app/helpers/orgs_helper.rb

```diff
@@ -1,19 +1,17 @@
 module OrgsHelper
+  # frozen_string_literal: true
```

Comment:
@briri @xsrust just want to make sure you're both familiar with this macro. [It's quite useful to use in most cases](https://freelancing-gods.com/2017/07/27/friendly-frozen-string-literals.html) :)

Reply:
Cool, I was not familiar with that.

Reply:
nor was I